### PR TITLE
Option to change interaction mode without notification

### PR DESCRIPTION
**Pull request by John-Doom**
Necessary for my mod: https://forums.dfworkshop.net/viewtopic.php?t=5373.

### CODE REVIEW

File: Assets/Scripts/Game/PlayerActivate.cs [hunk 1]
```diff
@@ -1400,7 +1400,7 @@ public void ChangeInteractionMode(PlayerActivateModes newMode)
             }
 
             // Present new mode to player
-            DaggerfallUI.SetMidScreenText(TextManager.Instance.GetLocalizedText("interactionIsNowInMode").Replace("%s", modeText));
+            if(!noText) DaggerfallUI.SetMidScreenText(TextManager.Instance.GetLocalizedText("interactionIsNowInMode").Replace("%s", modeText));
```
Comment (KABoissonneault) [inline Assets/Scripts/Game/PlayerActivate.cs:1403]: Could you put the `DaggerfallUI.SetMidScreenText(...)` on the next line (with proper indentation)?

**Review by KABoissonneault — APPROVED** [review]

File: Assets/Scripts/Game/PlayerActivate.cs (hunk 1 above)
Comment (John-Doom) [reply Assets/Scripts/Game/PlayerActivate.cs]: done :)

**Review by KABoissonneault — APPROVED** [review]

**Comment by Interkarma** [comment]
I'm know I'm being pedantic here, but can we please make this:

`public void ChangeInteractionMode(PlayerActivateModes newMode, bool showText = true)`

and

`if (showText)`

Then your mod can pass showText false to prevent text from being emitted. I just prefer this to `if (!noText)`.

Cheers. :)

**Comment by John-Doom** [comment]
Done :)

**Comment by Interkarma** [comment]
Cheers JD. :)

**Pull request merged**